src/pages/[platform]/start/migrate-to-gen2/index.mdx

```diff
@@ -196,10 +196,40 @@ The tables below present a feature matrix for Gen 1 customers who are considerin
 
 ### Other categories
 
+<InlineFilter filters={['android', 'swift']}>
+
+| Feature | Gen 1 | Gen 2 |
+|---|---|---| 
+| REST API| Yes|	Yes with custom CDK
```

Comment:
cc @tylerjroach is this accurate for android?

Reply:
No Rest is not supported on Android or Swift